integration-tests/cli/helpers/product-json2csv.data.js

```diff
@@ -0,0 +1,196 @@
+export const sampleProductType = {
```

Comment:
I would recommend making these functions in `createSampleProduct` otherwise you might end up with some wonky sharing of "references" of the modules. As ES6 modules export bindings live connections to values. Hope that makes sense and I am not wrong. Tried to dig up and old article [here](http://2ality.com/2015/07/es6-module-exports.html).

Reply:
Makes sense. But I don't think it will bw a problem in this particular case as side effects are not a problem here (each variable is used only once)